src/App.js

```diff
@@ -1,14 +1,28 @@
+/* eslint-disable import/extensions */
```

Comment:
- [ ] [REQUIRED] Good job so far! Please remove the `/* eslint-disable import/extensions */` comment and try to solve linter errors yourself, it will greatly help you to improve your experience 🙏

Reply:
Thank-you Sir for the guidance

Reply:
> * [x]  [REQUIRED] Good job so far! Please remove the `/* eslint-disable import/extensions */` comment and try to solve linter errors yourself, it will greatly help you to improve your experience 🙏

Noted with thanks